resources/EPvalidator.rb

```diff
@@ -498,6 +498,7 @@ def self.run_validator(hpxml_doc)
         "RelatedHVACSystem" => one, # HeatingSystem (boiler)
         "TankVolume" => one,
         "WaterHeaterInsulation/Jacket/JacketRValue" => zero_or_one, # Capable to model tank wrap insulation
+        "extension/StandbyLoss" => zero_or_one,
```

Comment:
Since this is an extension element and not (yet) defined in the HPXML schema, can you add a comment that defines the units and where the input can be obtained from?